crypto/secp256k1/private_test.go

```diff
@@ -128,7 +128,7 @@ func TestPrivateKeyFromHex(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := PrivateKeyFromHex(tt.args.hexkey)
+			got, err := PrivateKeyFromBytes([]byte(tt.args.hexkey))
```

Comment:
I think its best to remove this test as it is not needed. Please remove `func TestPrivateKeyFromHex` as `PrivateKeyFromBytes` is already tested elsewhere

Reply:
will do

Reply:
Removed `TestPrivateKeyFromHex` from secp256k1/private_test.go